crawl-ref/source/duration-data.h

```diff
@@ -640,6 +640,9 @@ static const duration_def duration_data[] =
     { DUR_BEOGH_CAN_RECRUIT, LIGHTBLUE, "Recruit", "", "can recruit",
       "You may recruit a defeated apostle into your service", D_EXPIRES,
        {{ "", end_beogh_recruit_window}}},
+    //TODO this needs rewrite
+    { DUR_GASTRONOMIC, YELLOW, "Gut", "transposing a stomach", "gastronomic expanse",
```

Comment:
As much as I could infer any consistent logic for different status light colours, YELLOW is generally used for maluses of a less serious nature. Various colours are used for magical effects but I would go with MAGENTA here (same as toxic radiance).

Reply:
Thanks, corrected